articles/vmware/vmw-gs.md

```diff
@@ -51,7 +51,7 @@ The first thing you need to do to get started with UKCloud for VMware is to log
 1. Go to the appropriate URL, depending on the security domain in which your service is located:
 
     - **Assured OFFICIAL:** <https://portal.ukcloud.com>
-    - **Elevated OFFICIAL:** <https://portal.ukcloud.gsi.gov.uk>
+    - **Elevated OFFICIAL:** Contact your Portal administrator or Customer Success Manager (CSM)
```

Comment:
Full stop at the end

Reply:
No need for bullet points to end in full stops if they are single, short sentences. As long as all bullets within a list are consistent.